src/views/studio/EditVideoSheet/EditVideoSheet.tsx

```diff
@@ -203,18 +201,19 @@ export const EditVideoSheet: React.FC = () => {
     }
 
     const refetchDataAndCacheAssets = async (videoId: VideoId) => {
-      const fetchedVideo = await refetchVideo({ where: { id: videoId } })
-
+      const fetchedVideo = await refetchVideoWithCursor({ where: { id_in: [videoId] } })
```

Comment:
I think better pattern for when we need to fetch something conditionally and only once is to do something like we currently have in  `CreateMemberView`:
```ts
await client.query<GetMembershipQuery, GetMembershipQueryVariables>({
  query: GetMembershipDocument,
  variables: { where: { handle: value } },
})
```